utils/terrascript_client.py

```diff
@@ -1302,6 +1318,13 @@ def populate_tf_resource_sqs(self, resource, namespace_info):
                     }
                 ]
             }
+            if len(kms_keys) > 0:
+                kms_statement = {
+                    "Effect": "Allow",
+                    "Action": ["kms:Decrypt"],
+                    "Resource": list(set(kms_keys))
```

Comment:
Why don't you make  `kms_keys` a set to begin with? That would make one less type conversion.